src/aerialmap_display.cpp

```diff
@@ -636,15 +636,17 @@ void AerialMapDisplay::transformTileToMapFrame()
     return;
   }
 
-  auto const center_tile = center_tile_->coord;
+  // FIXME: note the <double> template! this is different from center_tile_.coord, otherwise we could just use that
+  // since center_tile_ and ref_fix_ are in sync
+  auto const ref_fix_tile_coords = fromWGSCoordinate<double>({ ref_fix_->latitude, ref_fix_->longitude }, zoom_);
```

Comment:
@schra this template magic is really nasty here :/

Reply:
could be one way. main thing should be to differentiate between tile id, tile center coordinate (both are the same right?) and the actual tile coordinate